### PR TITLE
Fixing issue: Mobile view has padding on the right #563

### PR DESCRIPTION
**Pull request by Pathholder1806**
Hey @jtfairbank !

So as you suggested that the dimensions of the **react-globe** should be dynamic.

So I came up with a solution as described below:
1. I have taken the **dimensions of the globe** as a react state because we need the re-renders whenever the screen size changes so that **the globe object is re-rendered with the new appropriate width**.
2. I have used the an 'event listener' on the window object which is listening to the 'resize' and as a callback for it I have passed a function which updates the **globe dimension state** which apparently passes new width to the globe component and causes it to re-render with new width.

So now whenever the width of the parent container of **react-globe** changes, the width of **react-globe** also changes and the state updating causes its re-render.
Whenever the width of parent is more than 384(default), the width of **react-globe** is set to 384 and when the parent shrinks below 384px, the **react-globe** width is set to the full-width of the parent.

I am creating the pull request kindly review my changes and suggest any improvements you feel!

### CODE REVIEW

**Comment by Pathholder1806** [comment]
Hey @jtfairbank !

The is the selector in the index.css, **.link** that is causing errors in my local system

image

Now I don't know if is this is a problem with my local setup only or is it a actual error of the code.

**Comment by coderbyheart** [comment]
Preview: https://deploy-preview-579--da-landing-2.netlify.app/

File: src/components/vis/shipments-on-globe.tsx [hunk 1]
```diff
@@ -71,7 +71,8 @@ function buildGlobeVisData(lineItems: LineItem[]) {
 const ShipmentsOnGlobeVis: FC<Props> = ({ categoryVisItems }) => {
   const globeEl = useRef() as any
   const arcsData = buildGlobeVisData(categoryVisItems.nodes) as any
-  console.log('arcsData', arcsData)
+  // console.log('arcsData', arcsData)
```
Comment (coderbyheart) [inline src/components/vis/shipments-on-globe.tsx]:
```suggestion
```

File: src/components/vis/shipments-on-globe.tsx [hunk 2]
```diff
@@ -93,8 +94,34 @@ const ShipmentsOnGlobeVis: FC<Props> = ({ categoryVisItems }) => {
     globeEl.current.pointOfView({ lat: avgLat, lng: avgLng, altitude: 2.5 })
   }, [])
 
+  const [globeDimensions, setGlobeDimensions] = useState(384)
+
+  let resizeGlobeSize = () => {
```
Comment (coderbyheart) [inline src/components/vis/shipments-on-globe.tsx]:
```suggestion
  const resizeGlobeSize = () => {
```

File: src/components/vis/shipments-on-globe.tsx [hunk 3]
```diff
@@ -93,8 +94,34 @@ const ShipmentsOnGlobeVis: FC<Props> = ({ categoryVisItems }) => {
...
+    let globeSectionWidth =
```
Comment (coderbyheart) [inline src/components/vis/shipments-on-globe.tsx]:
```suggestion
    const globeSectionWidth =
```

File: src/stylesheets/index.css [hunk 4]
```diff
@@ -42,9 +42,9 @@
   width: 298px;
 }
 
-.link {
-  @apply text-navy-700 hover:underline font-semibold;
-}
+/* .link {
+  @apply text-navy-700 hover: underline font-semibold;
+} */
```
Comment (coderbyheart) [inline src/stylesheets/index.css]:
```suggestion
.link {
  @apply text-navy-700 hover: underline font-semibold;
}
```

Unrelated change, please address in separate issue.

**Review by coderbyheart — CHANGES_REQUESTED** [review]

**Review by coderbyheart — COMMENTED** [review]
Otherwise good solution!

**Comment by Pathholder1806** [comment]
Thanks @coderbyheart !
Although some checks failing.

File: src/stylesheets/index.css [hunk 5]
```diff
@@ -43,7 +43,7 @@
 }
 
 .link {
-  @apply text-navy-700 hover:underline font-semibold;
+  @apply text-navy-700 hover: underline font-semibold;
```
Comment (coderbyheart) [inline src/stylesheets/index.css]:
```suggestion
  @apply text-navy-700 hover:underline font-semibold;
```

**Comment by jtfairbank** [comment]
Resolves #563 (just adding this to link to the issue).  Thanks for the help @Pathholder1806!  Feel free to pick up another issue if you want to keep contributing. :smile:

**Pull request merged**